services/invitations/README.md

```diff
@@ -7,11 +7,2786 @@ Users invited via this Invitation Manager (libre graph API) will have `userType=
+      "name": "Webauthn Register Passwordless",
+      "providerId": "webauthn-register-passwordless",
+      "enabled": true,
+      "defaultAction": false,
+      "priority": 80,
+      "config": {}
+    },
+    {
+      "alias": "update_user_locale",
+      "name": "Update User Locale",
+      "providerId": "update_user_locale",
+      "enabled": true,
+      "defaultAction": false,
+      "priority": 1000,
+      "config": {}
+    }
+  ],
+  "browserFlow": "browser",
+  "registrationFlow": "registration",
+  "directGrantFlow": "direct grant",
+  "resetCredentialsFlow": "reset credentials",
+  "clientAuthenticationFlow": "clients",
+  "dockerAuthenticationFlow": "docker auth",
+  "attributes": {
+    "cibaBackchannelTokenDeliveryMode": "poll",
+    "cibaAuthRequestedUserHint": "login_hint",
+    "oauth2DevicePollingInterval": "5",
+    "clientOfflineSessionMaxLifespan": "0",
+    "clientSessionIdleTimeout": "0",
+    "clientOfflineSessionIdleTimeout": "0",
+    "cibaInterval": "5",
+    "realmReusableOtpCode": "false",
+    "cibaExpiresIn": "120",
+    "oauth2DeviceCodeLifespan": "600",
+    "parRequestUriLifespan": "60",
+    "clientSessionMaxLifespan": "0",
+    "adminEventsExpiration": ""
+  },
+  "keycloakVersion": "21.0.1",
+  "userManagedAccessAllowed": false,
+  "clientProfiles": {
+    "profiles": []
+  },
+  "clientPolicies": {
+    "policies": []
+  }
+}
+```
+
+Importing this into keycloak will give you a realm that federates with an LDAP server, has the right
+clients configured, and all the mappers correctly set. Be sure to set all the credentials after the import,
+as they will be disabled.
+
+The most relevant bits here are the mappers for the `OWNCLOUD_ID` and `OWNCLOUD_USER_TYPE` user properties.
 
+#### Backend configuration
 
-## Bridging provisioning delay
+After keycloak has been configured, you need to configure the backend with the following environment variables:
 
-When a guest account has to be provisioned in an external user management there might be a delay between creating the user and it being available in the local ocis system. In the first iteration the invitations service will only keep track of invites in memory. This list could be persisted in future iterations.
+* `INVITATIONS_KEYCLOAK_BASE_PATH`: The URL on which Keycloak is available.
+* `INVITATIONS_KEYCLOAK_CLIENT_ID`: The client ID of the client to use, in the above example, this would be `invitations`.
+* `INVITATIONS_KEYCLOAK_CLIENT_SECRET`: The client secret to use to authenticate, you can find this in the keycloak UI.
+* `INVITATIONS_KEYCLOAK_CLIENT_REALM`: The realm where the client was added, in the example above it would be `ocis`
+* `INVITATIONS_KEYCLOAK_USER_REALM`: The realm where to add the users, in the example above it would also be `ocis`.
+* `INVITATIONS_KEYCLOAK_INSECURE_SKIP_VERIFY`: If set to true, it will skip the verification of the keycloak https certificate. Not recommended in production enviroments.
```

Comment:
```suggestion
* `INVITATIONS_KEYCLOAK_BASE_PATH`: The URL to access Keycloak.
* `INVITATIONS_KEYCLOAK_CLIENT_ID`: The client ID of the client to use. In the above example, `invitations` is used.
* `INVITATIONS_KEYCLOAK_CLIENT_SECRET`: The client secret used to authenticate. This can be found in the Keycloak UI.
* `INVITATIONS_KEYCLOAK_CLIENT_REALM`: The realm where the client was added. In the example above, `ocis` is used.
* `INVITATIONS_KEYCLOAK_USER_REALM`: The realm where to add the users. In the example above, `ocis` is used.
* `INVITATIONS_KEYCLOAK_INSECURE_SKIP_VERIFY`: If set to true, the verification of the Keycloak https certificate is skipped. This is not recommended in production enviroments.
```